pkg/inventory/inventory-client.go

```diff
@@ -382,6 +387,22 @@ func (cic *ClusterInventoryClient) DeleteInventoryObj(info *resource.Info) error
 	return err
 }
 
+// ClearInventoryObj finds the inventory object in the list of objects,
```

Comment:
Are we passing list of objects to this method ?

Reply:
No. A legacy comment that is now out of date. Fixed.